code/api/src/modules/user/resolvers.js

```diff
@@ -43,21 +43,35 @@ export async function login(parentValue, { email, password }) {
       // Incorrect password
       throw new Error(`Sorry, the password you entered is incorrect. Please try again.`)
     } else {
-      const userDetailsToken = {
+      const userDetailsToken = { // Creates a new token for user logging in to store on server for future use.
         id: userDetails.id,
         name: userDetails.name,
         email: userDetails.email,
         role: userDetails.role
       }
 
-      return {
+      return { // Returns the details of the user and stores token made above
         user: userDetails,
         token: jwt.sign(userDetailsToken, serverConfig.secret)
       }
     }
   }
 }
 
+// set style
+export async function updateStyle(parentValue, { style, id }) {
+  if (style !== "") {
+    return await models.User.update(
+      {
+        style
+      },
+      { where: { id } }
+    )
+  } else {
+    throw new Error(`You must enter a style to set a user's style`)
+  }
+}
+
```

Comment:
This work around for now will allow for returning non-null response:

`export async function updateStyle(parentValue, { style, id }) {
  if (style !== "") {
    await models.User.update(
      {
        style
      },
      { where: { id } }
    );
    return await models.User.findOne({ where: { id } })
  } else {
    throw new Error(`You must enter a style to set a user's style`)
  }
}`